scripts/convert_to_uvfits.py

```diff
@@ -31,14 +31,18 @@
 for filename in args.files:
 
     # check output
-    outfilename = filename + '.uvfits'
+    splitext = os.path.splitext(filename)[1]
+    if splitext[1] == '.uvh5':
+        outfilename = splitext[0] + '.uvfits'
+    else:
```

Comment:
might be good to add some other extension conversions here. Measurement sets are always(?) denoted with `.ms`, FHD files end with `.sav`. Not sure there is a standard for Miriad, so we can leave that alone